### PR TITLE
translations: translate form options.

### PR DESCRIPTION
**Pull request by sebastiendeleze**
* Removes useless commit message template
* Adds translations extractor for JSON files.
* Moves schema http endpoint from REST application to UI.
* Adds form options for enum in JSON schemas.
* Adds a configuration for listing JSON keys to extract.
* Renames enum values with underscores instead of spaces or hyphens.
* Adds creator role for contributors during deposit publication.
* Removes "text-uppercase" CSS class for badge content.
* Removes useless translations markers in manual translations.
* Updates translations for enum values.

Co-Authored-by: Sébastien Délèze <sebastien.deleze@rero.ch>

### CODE REVIEW

File: tests/ui/test_views.py [hunk 1]
```diff
@@ -72,3 +72,12 @@ def test_logged_user(app, client, admin_user_fixture_with_db):
     res = client.get(url + '?resolve=1')
     assert b'"email":"admin@test.com"' in res.data
     assert b'"pid":"org"' in res.data
+
+
+def test_schemas(client):
+    """Test JSON schemas endpoint."""
+    res = client.get('/schemas/documents')
```
Comment (jma) [inline tests/ui/test_views.py:79]: can you use url_for instead?

File: tests/ui/test_views.py [hunk 2]
```diff
@@ -72,3 +72,12 @@ def test_logged_user(app, client, admin_user_fixture_with_db):
...
+    assert res.status_code == 200
+
+    res = client.get('/schemas/not_existing')
```
Comment (jma) [inline tests/ui/test_views.py:82]: url_for?

Comment (sebastiendeleze) [reply tests/ui/test_views.py]: done

File: tests/ui/test_views.py (hunk 1 above)
Comment (sebastiendeleze) [reply tests/ui/test_views.py]: done

**Pull request merged**